src/learn.jl

```diff
@@ -24,19 +24,45 @@ function LearnLogger(path, run_name; kwargs...)
     return LearnLogger(path, tensorboard_logger, Dict{String,Any}())
 end
 
+"""
+    log_event!(logger, value::AbstractString)
+
+Logs a string event given by `value` to `logger`.
+"""
+log_event!(logger, value)
+
 function log_event!(logger::LearnLogger, value)
```

Comment:
My thought is maybe we could do something like

```julia
function log_event!(logger, value)
```
Because LWB is dispatching this for a different type of logger, and it makes sense for lighthouse to provide the fallback methods for the different logging actions

Reply:
added fallback!